runtime_test.go

```diff
@@ -0,0 +1,47 @@
+package nullable
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/go-openapi/runtime"
+)
+
+func TestInt16ImplementsRuntimeValidatable(t *testing.T) {
```

Comment:
We could potentially also resort to the way e.g. the `pq` Postgres driver ensures the types implement the `sql.Driver` interface properly: https://github.com/lib/pq/blob/master/conn.go#L42
Could be added to each one of the generated files instead of having a runtime test for extra "security".

Reply:
agreed, rm'ed + added in each generated file.